inc/rest-api.php

```diff
@@ -61,3 +62,21 @@ function restrict_anonymous_rest_api_access( $response, array $handler, WP_REST_
 
 	return $response;
 }
+
+/**
+ * Enable Two Factor plugin routes to be accessed without elevated authentication.
+ *
+ * 2FA endpoint has to be accessible to subscriber users in order for them to
+ * gain elevated permissions elsewhere. Without this 2FA credentials cannot
+ * be set or updated.
+ *
+ * @param bool            $is_allowed Whether the endpoint is publicly accessible, false by default.
+ * @param WP_REST_Request $request    Active REST Request object.
+ * @return bool Whether the anonymous request should be permitted.
+ */
+function permit_2fa_endpoint_access( bool $is_allowed, WP_REST_Request $request ) : bool {
```

Comment:
```suggestion
function permit_2fa_endpoint_access( bool $is_allowed, WP_REST_Request $request ): bool {
```
Per linter CI action's request